lib/matplotlib/axis.py

```diff
@@ -2385,8 +2385,13 @@ def _update_offset_text_position(self, bboxes, bboxes2):
         Update the offset_text position based on the sequence of bounding
         boxes of all the ticklabels
         """
-        x, y = self.offsetText.get_position()
-        top = self.axes.bbox.ymax
+        x, _ = self.offsetText.get_position()
+        if 'outline' in self.axes.spines:
+            # Special case for colorbars:
+            bbox = self.axes.spines['outline'].get_window_extent()
```

Comment:
Coverage is complaining about this line, and that seems to be true, as no test hits this (not even the new ones). So do we need this change at all?

Reply:
Ah sorry, I was having issues with warnings and accidentally stashed the exception I was using to check if this was hit.

So this line change does have an effect, and I'm not sure why coverage is complaining.